db/migrate/20250722205209_create_task_completions.rb

```diff
@@ -0,0 +1,12 @@
+class CreateTaskCompletions < ActiveRecord::Migration[8.0]
+  def change
+    create_table :task_completions do |t|
+      t.references :task, null: false, foreign_key: { on_delete: :cascade }
```

Comment:
Question: does this mean if the TASK is completed, delete the COMPLETIONS (and not the other way around that would be bad?)